### ISSUE TITLE
Kafka and fullnameOverride value

### ISSUE THREAD

**Issue by mkmik** [bitnami/charts#998]
TL;DR please add fullnameOverride and relevant template changes in _helper.tpl, see https://github.com/helm/helm/issues/1186

**Comment by carrodher** [comment]
Got it. Thanks, @mkmik, we will take a look.

**Comment by tompizmor** [comment]
@mkmik we have added the fullnameOverride in the chart version `1.2.4`

**Comment by tompizmor** [comment]
@mkmik I am closing this issue as fullnameOverride was already added. Please re-open the issue if you find any problem with it.
